Tasks/Common/nuget-task-common/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -12,5 +12,6 @@
   "loc.messages.Error_NoVersionWasFoundWhichMatches": "No version was found which matches the input %s",
   "loc.messages.Error_NoUrlWasFoundWhichMatches": "No download URL was found for %s",
   "loc.messages.Error_NuGetToolInstallerFailer": "Tool install failed: %s",
-  "loc.messages.Warning_UpdatingNuGetVersion": "Updating version of NuGet.exe to %s from %s. Behavior changes or breaking changes might occur as NuGet updates to a new version. If this is not desired, uncheck the 'Check for Latest Version' option in the task."
+  "loc.messages.Warning_UpdatingNuGetVersion": "Updating version of NuGet.exe to %s from %s. Behavior changes or breaking changes might occur as NuGet updates to a new version. If this is not desired, uncheck the 'Check for Latest Version' option in the task.",
+  "loc.messages.Error_NoMatchingFilesFoundForPattern": "No matching files were found with search pattern: %s"
```

Comment:
This change should go into nuget-task-common/module.json, and then on build will also get added to this autogenerated file.

Reply:
Also... the resource strings defined in nuget-task-common are not actually used -- the resources are actually loaded from the build task task.json files. If you want, you can omit the resource change, I have a PR coming to fix up all the resource strings, since there are a few omissions.

Reply:
These keep coming in and out on build, not sure why. Happy to let your PR go in with all the fixes first.